cfgen/cfgen

```diff
@@ -783,6 +783,16 @@ Oid.__doc__ = 'Motr conf object identifier'
 Oid.fidk.__doc__ = '.f_key part of the corresponding m0_fid'
 
 
+ObjTMask = NamedTuple('ObjTMask', [('cont', int), ('key', int)])
+ObjTMask.__doc__ = 'Motr conf object identifier mask'
+ObjTMask.key.__doc__ = '.f_key part mask of the corresponding m0_fid'
+
+
+ObjT_Mask_Map = {
+    ObjT.process: ObjTMask(0xffffffffffffffff, 0xffffffff00000000)
```

Comment:
I think it would be nice if we make use of some macros something like
QW_F (for 16 - Fs)
QW_0 (For 16 - 0s)
DW_F (For 8 - Fs)
DW_0 (For 8 - 0s)

BSH_DW = 8x4
When we want to make FFFFFFFF00000000, we can write something like (DW_F < x_bits) using left shift operation.
DW_F_DW_0 = DW_F < BSH_DW

Abbreviations are :
QW = QuadWord
DW = DoubleWord
BSH_DW = DWord Bit Shifter

Reply:
Done, thanks. Please check